lib/context.js

```diff
@@ -20,15 +20,21 @@ async function refresh() {
   const client = await getClient(config);
   const oldTokenSet = tokenSet.call(this);
   const newTokenSet = await client.refresh(oldTokenSet);
-  // If no new refresh token assume the current refresh token is valid.
-  if (!newTokenSet.refresh_token) {
-    newTokenSet.refresh_token = oldTokenSet.refresh_token;
-  }
 
-  // Update the session's tokenSet
+  // Update the session
   const session = req[config.session.name];
+  Object.assign(session, {
```

Comment:
Fixed issue where refreshed ATs were not being persisted in the session.

I'm now updating `req[config.session.name]` rather than the `cachedTokenSet`

Test case https://github.com/auth0/express-openid-connect/pull/127/files#diff-1840f764728973b9eb715f3e751c1978